draft/2022-08-31-this-week-in-rust.md

```diff
@@ -37,6 +37,8 @@ and just ask the editors to select the category.
 * [rust-analyzer - changelog #144](https://rust-analyzer.github.io/thisweek/2022/08/29/changelog-144.html)
 * [IntelliJ Rust Changelog #177](https://intellij-rust.github.io/2022/08/29/changelog-177.html)
 * [Diesel 2.0.0](https://diesel.rs/news/2_0_0_release.html)
+* [Helix Editor Release 22.08 Highlights](https://helix-editor.com/news/release-22-08-highlights/)
```

Comment:
As a heads up: We also got this addition via #3612 which I already applied - I will make sure we don't duplicate it before I merge.